dts/msm8916/msm8916-coolpad-coolpady80d.dts

```diff
@@ -0,0 +1,45 @@
+//SPDX-License-Identifier: GPL-2.0-only
+/*
+Notes regarding the device 'coolpad-coolpady80d':
+
+This device comes with a stock ROM based on Android 4.4 and 1GB RAM.
+Therefore, it is considered quirky, and lk1st should be mandatory for mainline to run on this device.
+The commented out "yulong," lines are what I left behind initially trying to port lk2nd to the device.
+Those are kept in the DTS for reference purposes only.
+
+Regarding the device's DT:
+No kernel sources are availble for this device, so I have tried decompiling the device tree extracted from
+a known working boot.img file.
+However, I attempted using multiple QCDT extractors and they all came back with output that made dtc complain
+that it has bad magic. Taking how only an lk2nd-patched known working bootimg would be able to make the stock
+bootloader boot into lk2nd (not even lk2nd-appended-dtb.img would boot) into account, the stock bootloader might
+be expecting extra magic before the valid device tree data. If that is the case, this would effectively stop
+any custom-built kernel from booting at all as long as the stock bootloader still exists in the boot process.
+
+TODO: I may consider looking into the panel's DTSI generated by lmdpdg to get the panel to work in lk1st.
+*/
+
+/dts-v1/;
+
+#include <skeleton.dtsi>
+
+/ {
+
+	qcom,msm-id = <206 0>;
+	qcom,board-id = <8 0>;
```

Comment:
https://github.com/msm8916-mainline/lk2nd/blob/606020a090f38b8bb28acc82106f64f6b5587f96/dts/msm8916/msm8916-mtp.dts#L11-L12

Duplicated id due to recent changes on branch `master`. This device tree is supposed to be merged into `msm8916-mtp` now.

Reply:
So I am supposed to deprecate this file and instead add panel code in `msm8916-mtp.dts`?

Reply:
In fact, was supposed from the start, since 206/8 is the id for MTP